pkg/applier/applier_test.go

```diff
@@ -915,6 +915,7 @@ func TestProcessApplyEvent(t *testing.T) {
 	deploymentObj := newDeploymentObj()
```

Comment:
as a person who has never seen this test before I find it a little hard to figure out what the intent of this is.  Maybe a little context would help.  The comments as they stand seem to just outline what is happening next

Reply:
Could you clarify which intent you'd like to see more context on?